src/Components/ArticleComments.jsx

```diff
@@ -22,9 +34,19 @@ export default function ArticleComments({
             <li key={comment.comment_id} className="comments">
               <span className="comment-user-name">{comment.author} </span>
               <span className="comment-time">{comment.created_at}</span>
-              <br></br>
               <span className="comment-comment">{comment.body}</span>
-              <p className="comment-vote">Votes: {comment.votes}</p>
+
+              <span className="comment-vote">Votes: {comment.votes}</span>
+              {loggedInUser.username === comment.author ? (
+                <button
```

Comment:
This conditional rendering of the button if it's an author that matches the logged in user is great ✅